src/app/Shared/Services/CommandChannel.service.tsx

```diff
@@ -16,26 +17,26 @@ export class CommandChannel {
   private readonly grafanaDatasourceUrlSubject = new ReplaySubject<string>(1);
   private pingTimer: number = -1;
 
-  constructor(apiSvc: ApiService) {
+  constructor(apiSvc: ApiService, private readonly notifications: Notifications) {
     this.apiSvc = apiSvc;
 
     fromFetch(`${this.apiSvc.authority}/clienturl`)
       .pipe(concatMap(resp => from(resp.json())))
       .subscribe(
         (url: any) => this.clientUrlSubject.next(url.clientUrl),
-        (err: any) => console.error('Client URL configuration', err) // TODO add toast notification
+        (err: any) => this.logError('Client URL configuration', err)
       );
 
     fromFetch(`${this.apiSvc.authority}/grafana_datasource_url`)
       .pipe(concatMap(resp => from(resp.json())))
       .subscribe(
         (url: any) => this.grafanaDatasourceUrlSubject.next(url.grafanaDatasourceUrl),
-        (err: any) => console.error('Grafana Datasource configuration', err) // TODO add toast notification
+        (err: any) => this.logError('Grafana Datasource configuration', err)
```

Comment:
In my test I see the error as just: 

Danger alert:Grafana Datasource configuration
{}

Is the empty brackets expected, or just to do with how I'm running it through cjfr?

Reply:
That's the normal result right now. The response has a 500 status code but no payload, so that's what you see in the toast notification currently (unfortunately).

Reply:
Okay, maybe we can open an issue to target the error notifications in general, seems some need cross-project work. Then we can resolve them later. What do you think?

Reply:
Sounds good to me.

Reply:
Okay, let's do that. Can you resolve this after an issue is created?

Reply:
#65